src/test/java/bio/terra/service/resourcemanagement/google/GoogleResourceDaoUnitTest.java

```diff
@@ -98,45 +94,44 @@ public void teardown() {
   }
 
   /* Helper method to create a minimal dataset and register its ID for cleanup */
-  private Dataset createDataset(UUID projectResourceId) throws IOException {
+  private void createDataset(UUID projectResourceId) throws IOException {
     Dataset dataset =
         daoOperations.createDataset(
             billingProfile.getId(), projectResourceId, DaoOperations.DATASET_MINIMAL);
     datasetIds.add(dataset.getId());
-    return dataset;
   }
 
   @Test
-  public void twoDatasetsTwoBillingProfilesTwoBuckets() {
+  void twoDatasetsTwoBillingProfilesTwoBuckets() {
     List<GoogleProjectResource> retrievedProjects =
         googleResourceDao.retrieveProjectsByBillingProfileId(billingProfile.getId());
 
     assertThat("Project Count should be 2", retrievedProjects, hasSize(2));
     retrievedProjects.forEach(
         project ->
             assertFalse(
-                "Projects by default use the general TDR SA",
-                project.hasDedicatedServiceAccount()));
+                project.hasDedicatedServiceAccount(),
+                "Projects by default use the general TDR SA"));
 
     UUID projectId1 = projectResourceIds.get(0);
     UUID projectId2 = projectResourceIds.get(1);
     String dedicatedSa = "dedicated-sa@gmail.com";
     googleResourceDao.updateProjectResourceServiceAccount(projectId1, dedicatedSa);
-    assertTrue(
+    assertThat(
         "Dedicated service account is detected",
         googleResourceDao.retrieveProjectById(projectId1).hasDedicatedServiceAccount());
-    assertFalse(
+    assertThat(
         "Unaltered project still uses the general TDR SA",
-        googleResourceDao.retrieveProjectById(projectId2).hasDedicatedServiceAccount());
+        !googleResourceDao.retrieveProjectById(projectId2).hasDedicatedServiceAccount());
```

Comment:
Earlier `assertFalse` is used, but here `assertThat` is used. When should one be used versus another? I think `assertFalse` or `assertTrue` is clearer, generally speaking.